composer.json

```diff
@@ -11,17 +11,19 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
-    },
     "require": {
         "php": ">=5.5",
         "ext-hash": "*",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "ext-curl": "*"
+        "ext-curl": "*",
+        "composer/ca-bundle": "~1.0",
+        "symfony/options-resolver": "~2.8|~3.0"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "~2.1",
+        "phpunit/phpunit": "^4.8 || ^5.0",
+        "monolog/monolog": "*"
```

Comment:
>I would advise against moving this block of code

I agree with you, this change should not be here. Thank you pointing it out

>I can work on a PR to address all this issues with dependencies.

It would be great, but I'm not sure this is the proper time to do it, because I have a lot of other changes to do on features that will drop the dependencies or will to update them, so it could be that you just lose your time fixing something that soon or later could not even be there anymore